atomic_reactor/plugins/pre_add_help.py

```diff
@@ -66,6 +69,33 @@ def run(self):
             result['status'] = self.NO_HELP_FILE_FOUND
             return result
 
+        dockerfile = df_parser(self.workflow.builder.df_path, workflow=self.workflow)
+        labels = dockerfile.labels
+
+        with open(help_path, 'r+') as help_file:
+            try:
+                lines = help_file.readlines()
+
+                if not lines[0].startswith("% "):
+
+                    lines.insert(0, "%% %s (1) Container Image Pages\n" %
+                                 (get_preferred_label(labels, "name") or ""))
+                    lines.insert(1, "%% %s\n" %
+                                 (get_preferred_label(labels, "maintainer") or ""))
+                    lines.insert(2, "%% %s\n" % dt.fromtimestamp(atomic_reactor_start_time)
+                                 .strftime(format="%B %-d, %Y"))
+
+                help_file.seek(0)
+                help_file.truncate()
+                help_file.writelines(lines)
+
+                self.log.info("Added metadata to %s for generating nicer manpages." % help_path)
```

Comment:
We only need to write the file if we changed `lines`, so these four lines could be moved into the `if not ...:` block.

Reply:
you're right my fault i'll fix that

Reply:
Already done but GH doesn't want to mark it as outdated.